src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.model;
+
+public class Car {
+
+    private int distance;
+
+    public Car(String name) {
```

Comment:
```suggestion
    public Car() {
```
☠️ ☠️ ☠️ ☠️ ☠️ ☠️ ☠️ ☠️ ☠️ ☠️